integration-tests/src/test/java/com/ozonehis/eip/odoo/openmrs/ServiceRequestToSaleOrderIntegrationTest.java

```diff
@@ -122,8 +122,8 @@ public void shouldCreateSaleOrderInOdooGivenServiceRequestBundle() {
         assertNotNull(createdSaleOrder);
         assertEquals(ENCOUNTER_PART_OF_UUID, createdSaleOrder.getOrderClientOrderRef());
         assertEquals("draft", createdSaleOrder.getOrderState());
-        assertEquals("77.0 kg", createdSaleOrder.getPartnerWeight());
-        assertEquals("1984-01-01", createdSaleOrder.getPartnerBirthDate());
+        //        assertEquals("77.0 kg", createdSaleOrder.getPartnerWeight());
```

Comment:
Please add a comment on why this is commented. Add a Todo item so that we remember to uncomment later.

Reply:
I think better we revert this commit as soon as the release PR is in. WDYT?